bin/api-docs/gen-components-docs/markdown/index.mjs

```diff
@@ -8,14 +8,18 @@ import json2md from 'json2md';
  */
 import { generateMarkdownPropsJson } from './props.mjs';
 
+function normalizeTrailingNewline( str ) {
+	return str?.length ? str.replace( /\n*$/, '\n' ) : undefined;
```

Comment:
Just to be sure I understand, this aims to result in having exactly one newline, even if the description doesn't come with a newline, right? Even in the case of an empty description string, which will also result in just a newline?

Reply:
Good question. I hope this captures the intent better: e5e4a65fcd86234150a29a365187d6ed9cfaa908

Reply:
Feel free to ship 🚀